packages/vite/src/node/server/index.ts

```diff
@@ -508,7 +508,7 @@ export async function createServer(
   ws.on('vite:invalidate', async ({ path }: InvalidatePayload) => {
     const mod = moduleGraph.urlToModuleMap.get(path)
     if (mod && mod.isSelfAccepting && mod.lastHMRTimestamp > 0) {
-      const file = getShortName(mod.file!, config.root)
+      const file = getShortName(mod.file, config.root)
```

Comment:
**refactor:** Removing nullability of `ModuleNode.file` as discussed with @patak-dev